charts/linkerd2-cni/README.md

```diff
@@ -13,20 +13,20 @@ The following table lists the configurable parameters of the Linkerd2-cni chart
 
 | Parameter                            | Description                                                           | Default                       |
 |--------------------------------------|-----------------------------------------------------------------------|-------------------------------|
-|`namespace`                           | Control plane namespace                                               | `linkerd`|
-|`controllerNamespaceLabel`            | Control plane label. Do not edit                                      |`linkerd.io/control-plane-ns`|
+`cniPluginImage`                      | Docker image for the cni plugin                                       |`gcr.io/linkerd-io/cni-plugin`|
```

Comment:
Nit: Can we uppercase the `cni` used in the _Description_ (here and on line 17) to match the rest of the documentation? (I recognize this was introduced in a previous PR.) :pray:

Reply:
Updated.